proxy/src/auth/credentials.rs

```diff
@@ -61,7 +61,15 @@ impl<'a> ClientCredentials<'a> {
         // Project name might be passed via PG's command-line options.
         let project_option = params
```

Comment:
Do we want to rename it to `endpoint_option`?

Reply:
not yet, I intentionally didn't changed the whole code base